packages/app/components/stack-modal/StackProgress.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  OrderProps,
+  fundsUsedWithToken,
+  totalOrdersDone,
+  totalFundsUsed,
+} from "@/app/models/order";
+import { OrdersProgressBar } from "@/components/OrdersProgressBar";
+import { BodyText } from "@/ui";
+
+export const StackProgress = ({ order }: OrderProps) => (
+  <div className="mt-3 space-y-3">
+    <div className="flex flex-col justify-between space-y-1 md:space-y-0 md:items-center md:flex-row">
+      <OrdersExecuted order={order} />
+      <div className="flex items-center space-x-1">
+        <BodyText size="responsive" className="text-em-low">
+          Total funds used:{" "}
```

Comment:
```suggestion
          Total Funds used:{" "}
```

I like how the design looks :)

Reply:
I don't think that capitalizing funds make sense. I prefer: Total funds used:

Reply:
I think it's a reading hack, where you put more emphasis on what matters (which is Total Funds), but it works in both ways for me.

Reply:
I bite this. Updated.